src/Comparator.php

```diff
@@ -94,28 +94,18 @@ public static function notEqualTo($version1, $version2)
     }
 
     /**
-     * Evaluate $version1 $operator $version2.
+     * Evaluates the expression: $version1 $operator $version2.
      *
      * @param string $version1
-     * @param string $operator Comparison operator like ">" or "<=", etc.
```

Comment:
Because it should either list all operators possible then. I would prefer to just assume the user knows what a valid operator is. Though in most cases I suspect they will use the methods that actually call `compare()` and not call `compare()` directly. The operators can be derived from the function description of all other methods.